modules/ROOT/pages/enable-flow-sources-concept.adoc

```diff
@@ -1,82 +1,64 @@
-
-[source,xml,linenums]
+[source,xml,linenums,subs="verbatim,attributes"]
 ----
-
 <http:listener-config name="HTTP_Listener_config">
-		<http:listener-connection host="localhost" port="${http.transport.port}" />
+    <http:listener-connection host="localhost" port="{port}"/>
 </http:listener-config>
 
-<flow name="http-transportFlow">
-
-  <http:listener config-ref="HTTP_Listener_config" path="/"/>
-  <set-payload value='#["Hello World!"]'/>
-
-
+<flow name="{flow_name}">
+    <http:listener path="/" config-ref="HTTP_Listener_config"/>
+    <set-payload value="#[{payload}]" mimeType="text/plain"/>
 </flow>
 ----
 
-In order for MUnit to be able to reach the endpoint, you need to enable the http-transportFlow in your enable-flow-source property:
+For MUnit to reach the endpoint, you need to enable the `{flow_name}` in your `enable-flow-source` property:
 
-[source,xml,linenums]
+[source,xml,linenums,subs="verbatim,attributes"]
 ----
-
-<http:request-config name="HTTP_Request_configuration">
-  <http:request-connection host="localhost" port="${http.transport.port}" />
+<http:request-config name="HTTP_Request_config">
+    <http:request-connection host="localhost" port="{port}"/>
 </http:request-config>
 
-<munit:test name="http-request-test-validatePayload" description="test">
-
+<munit:test name="http-example-test">
     <munit:enable-flow-sources>
-        <munit:enable-flow-source value="http-transportFlow"/>
+        <munit:enable-flow-source value="{flow_name}"/> <!--1-->
     </munit:enable-flow-sources>
 
     <munit:execution>
-        <http:request method="GET" path="/" config-ref="HTTP_Request_configuration"/>
+        <http:request method="GET" path="/" config-ref="HTTP_Request_config"/>
     </munit:execution>
 
     <munit:validation>
-        <munit-tools:assert-that expression="#[payload]" is="#[MunitTools::equalTo('Hello World')]" />
+        <munit-tools:assert-that expression="#[payload]"
+                                 is="#[MunitTools::equalTo({payload})]"/>
     </munit:validation>
-
 </munit:test>
 ----
+<1> If the `{flow_name}` is not listed as an enabled flow source, your MUnit test fails because it cannot initialize the
+HTTP listener configuration.
 
-If the http-transportFlow is not listed as an enabled flow source, your MUnit test fails because it cannot initialize the http-listener configuration.
-
-== To Enable Flow Sources from Studio
+== Enabling Flow Sources from Studio
 
-You can also define flow sources using the Mule Properties view of your MUnit Test in Anypoint studio:
+You can also define flow sources using the *Mule Properties* view of your MUnit test in Anypoint Studio:
 
-image::enable-flow-sources-concept-7f313.png[]
+image::enable-flow-sources-concept.png[]
 
-. Click the green Add icon.
-. Select the name of the flow you want to enable from the Name Of The Flow drop-down
+. Click the green *Add* icon.
+. Select the name of the flow you want to enable from the *Name of the flow* drop-down menu.
 
```

Comment:
If the UI is not too cluttered, consider reducing references to the UI controls to their labels only:
 
     Click *Add.*
     Select the name of the flow you want to enable from *Name of the flow*.

GLOBAL consideration for all writing going forward.